docker-compose.yml

```diff
@@ -11,7 +11,8 @@ services:
       APPLICATION_ROOT: "/"
       PORT0: "${SPIFFWORKFLOW_FRONTEND_PORT:-8001}"
       SPIFFWORKFLOW_FRONTEND_RUNTIME_CONFIG_APP_ROUTING_STRATEGY: path_based
-      SPIFFWORKFLOW_FRONTEND_RUNTIME_CONFIG_BACKEND_BASE_URL: http://localhost:8003/api
+      SPIFFWORKFLOW_HOST: "${SPIFFWORKFLOW_SERVER_NAME:-localhost}"
+      SPIFFWORKFLOW_FRONTEND_RUNTIME_CONFIG_BACKEND_BASE_URL: "http://${SPIFFWORKFLOW_HOST}:8003/api"
```

Comment:
This one may need the `:-localhost` as well.

Reply:
fixed